docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -477,5 +477,65 @@ pattern:
         json:
           name: -o json
           description: display the content of a pattern in json format
-          usage: |
-              mesheryctl pattern view [pattern-name|pattern-id] -o json
\ No newline at end of file
+          usage:
+              mesheryctl pattern view [pattern-name|pattern-id] -o json
+
+app:
+  name: app
+  description: Service Mesh application management
+  usage:
+    mesheryctl app 
+  flags:
+    file:
+      name: --file, -f
+      description: (required) path to pattern file
+      usage:
+        mesheryctl app --file [path to pattern file]
+  subcommands:
+    list:
+      name: list
+      description: displays a list of available applications
+      usage:
+          mesheryctl app list
+    offboard:
+      name: offboard
+      description: offboard application
+      usage:
+          mesheryctl app offboard [app name] [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: Offboard application with an application file
+          usage:
+              mesheryctl app offboard -f [filepath] 
+    onboard:
+      name: onboard
+      description: onboard application
+      usage: 
+          mesheryctl app onboard [app name] [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: onboard application with application file
+          usage:
+              mesheryctl app onboard --file [filepath]
+    view:
+      name: view
+      description: displays the contents of a specific pattern file
+      usage: | 
+          mesheryctl app view BookInfo (application-name|application-id)
```

Comment:
hey, you beat me off Rog! I suggested you let's make it either one of them; similarly in the examples section; e.g. https://614acc4b159f7366ea1050ba--meshery-docs-preview.netlify.app/reference/mesheryctl/apps#examples
https://614acc4b159f7366ea1050ba--meshery-docs-preview.netlify.app/reference/mesheryctl/apps/onboard#examples

Reply:
WAIT... It's still there? 🤔 I corrected them 😤 ok I'll check again

Reply:
To do in Issue #3978

Reply:
For now, no examples. So I leave it for this PR